src/components/Guides.tsx

```diff
@@ -201,8 +201,8 @@ const genericGuide: any = {
 	) => ({
 		intro: renderGenericInfo(deviceType?.name),
 		steps: [
-			renderDownloadOS(sdk, app, deviceType),
 			renderDownloadEtcher(),
+			renderDownloadOS(sdk, app, deviceType),
```

Comment:
download os should be the first step, @chrisys updated this earlier today.

Reply:
I saw in the designs it's etcher :/ cool thanks